releng/org.eclipse.xtext.tycho.parent/pom.xml

```diff
@@ -25,8 +25,8 @@
 		<log4j.version>1.2.16</log4j.version>
 		<commons-logging.version>1.1.3</commons-logging.version>
 		<commons-cli.version>1.2</commons-cli.version>
-		<equinox-common-version>3.6.0</equinox-common-version>
-		<jdt-core-version>3.10.0</jdt-core-version>
+		<equinox-common-version>3.8.0</equinox-common-version>
+		<jdt-core-version>3.12.2</jdt-core-version>
```

Comment:
You mean we shouldn't allow the dependencies to diverge between Maven and the target platform? Maybe we could solve that by re-running the xtext-extras tests in the eclipse build using the target platform, which would verify whether our bundles are binary compatible with Eclipse Luna. That could run as an extra pipeline stage in the Jenkins build.